Source/dom/layers/__tests__/Text.test.js

```diff
@@ -78,3 +78,60 @@ test('should fix the width', () => {
 
   expect(text.fixedWidth).toBe(true)
 })
+
+test('should return the fragments of a text layer', () => {
+  let text = new Text({
+    text: 'blah',
+  })
+  text.adjustToFit()
```

Comment:
Totally unrelated to this PR, but I wonder if we should make the `text.adjustToFit()` bit part of the initialisation for `Text` objects? Or is that something that we don't want to do all the time?

Also, would it be nice / useful to support a simpler method of creating a Text layer? Something like `let text = new Text('blah')`?

Reply:
> if we should make the `text.adjustToFit()` bit part of the initialisation for Text objects? Or is that something that we don't want to do all the time?

I guess when the `frame` is not specified we could do it automatically. If it's specified, I don't think we should.

> Something like let `text = new Text('blah')`?

That'd be really easy to do but I'm not convinced it's worth it: it creates an asymmetry between layers, harder to document, increase the API surface, etc.. And it's not like you would write `new Text({ text: 'blah' })` that much, is it?